tsunamibayes/scenario.py

```diff
@@ -183,7 +193,7 @@ def save_data(self,output_dir,append_rows=None):
             self.bayes_data.iloc[n:].to_csv(output_dir+"/bayes_data.csv",mode='a+',header=False)
             self.debug.iloc[n:].to_csv(output_dir+"/debug.csv",mode='a+',header=False)
 
-    def sample(self,nsamples,output_dir=None,save_freq=1,verbose=False):
+    def sample(self,nsamples,output_dir=None,save_freq=10,verbose=False):
```

Comment:
The default value of save_freq is a bit ambiguous. See utils.py and see that it shows there that the default should be 10...I'm not sure if we were going for 10 or 1 here.

Reply:
I adjusted it to be `save_freq=1,` since we're now just appending to the files rather than re-writing them every time `save_data` is called.